src/lib/util/TransferUtil.ts

```diff
@@ -0,0 +1,71 @@
+import { AssetAdministrationShell, AssetKind, Submodel } from '@aas-core-works/aas-core3.0-typescript/types';
```

Comment:
I would move this file to the transfer-service folder since it is only used there, right?

Reply:
done